frame/ibc/Cargo.toml

```diff
@@ -35,10 +35,11 @@ sp-trie = { git = "https://github.com/paritytech/substrate", branch = "polkadot-
 xcm = { default-features = false, git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.22" }
 
 hex-literal = { version = "0.3.4", optional = true }
-ibc = { git = "https://github.com/ComposableFi/ibc-rs", rev = "b3ab319018b1bb8917bca3aad977455aa2c9b179", default-features = false, features = [
+ibc = { git = "https://github.com/ComposableFi/ibc-rs", branch = "feat/revert-tracing-version/andor0", default-features = false, features = [
```

Comment:
Couldn't we merge this into `ibc-rs` master branch, instead of depending on this branch here, in case we make any other changes to ibc-rs, we don't want to be checking out from this branch.